slugify: ModuleNotFoundError when running test cases.

**Issue Description**
slugify: ModuleNotFoundError when running test cases.

```
ImportError while loading conftest '/workspace/ds/dirsrvtests/conftest.py'.
dirsrvtests/conftest.py:12: in <module>
    from slugify import slugify
E   ModuleNotFoundError: No module named 'slugify'
```

**Package Version and Platform:**
- Platform: RHEL 9 (will check for fedora)

**Steps to Reproduce**
Steps to reproduce the behavior:
1. Run any test case

**Expected results**
Test cases should run without any error.

--- a/dirsrvtests/conftest.py
+++ b/dirsrvtests/conftest.py
@@ -18,8 +18,10 @@ import gzip
 from .report import getReport
 from lib389.paths import Paths
 from enum import Enum
-from slugify import slugify
-from pathlib import Path
+
+if "WEBUI" in os.environ:
+    from slugify import slugify
+    from pathlib import Path
 
 
 pkgs = ['389-ds-base', 'nss', 'nspr', 'openldap', 'cyrus-sasl']
@@ -139,7 +141,7 @@ def pytest_runtest_makereport(item, call):
             report.extra = extra
 
     # Make a screenshot if WebUI test fails
-    if call.when == "call":
+    if call.when == "call" and "WEBUI" in os.environ:
         if call.excinfo is not None and "page" in item.funcargs:
             page = item.funcargs["page"]
             screenshot_dir = Path(".playwright-screenshots")

